modules/motions/ui/MotionFormStartNew/Parts/StartNewSDVTTargetValidatorLimitsUpdate.tsx

```diff
@@ -45,11 +44,14 @@ export const formParts = createMotionFormPart({
         'tuple(uint256 nodeOperatorId, bool isTargetLimitActive, uint256 targetLimit)[]',
       ],
       [
-        sortedNodeOperators.map(nodeOperator => ({
-          nodeOperatorId: Number(nodeOperator.id),
-          isTargetLimitActive: nodeOperator.isTargetLimitActive,
-          targetLimit: Number(nodeOperator.targetLimit),
-        })),
+        sortedNodeOperators.map(nodeOperator => {
+          const targetLimit = Number(nodeOperator.targetLimit)
+          return {
+            nodeOperatorId: Number(nodeOperator.id),
+            isTargetLimitActive: targetLimit === 0 ? false : true,
```

Comment:
I see that you get rid from isTargetLimitActive is it a part of agreements?

Reply:
Yes, we discussed it with Anna and decided to get rid of the checkbox, since it's causing some confusion in terms of UX. We can for sure say that setting limit to 0 is an equivalent of setting `isTargetLimitActive` to `false`. Wdyt?

Reply:
I like the solution in general, but it seems to me that the user needs to be informed about this somehow. Since this causes some hidden actions:
1) enable the limit with a positive value
2) disable the limit at zero value
This can be added to a hint, tooltip or documentation. Describe the option that you like best.
If it's not too difficult I would suggest making information at Motion page rather documentation.